django_airavata/static/common/scss/main.scss

```diff
@@ -117,6 +117,7 @@ body {
     flex-grow: 1;
     padding: 1rem;
     max-width: 1200px;
+    width: 100%;
```

Comment:
Can you please explain why you added this? I don't see what this is doing.

Reply:
This was added to make `main-content` with 100% or 1200px max. Otherwise child element cannot be wrapped to the full width.

Reply:
Specifically, can you describe where in the UI this makes a difference? For example, when a file is being edited in the codemirror editor?  I just want to verify if this is the best fix and also see if this change impacts any other views in the portal.

Reply:
This is not needed. Removed